.lintrunner.toml

```diff
@@ -300,7 +309,11 @@ command = [
 [[linter]]
 code = 'NOQA'
 include_patterns = ['**/*.py', '**/*.pyi']
-exclude_patterns = ['caffe2/**']
+exclude_patterns = [
+    'caffe2/**',
+    'fb/**',
+    '**/fb/**'
```

Comment:
I see various combinations of these three patterns added to all the linters.

1. Why do the exclusions added vary?  Aren't they supposed to exclude the same set of folders?
2. Any reason to not add this exclusion pattern to every linter?  That'll make them more consistent and give anyone adding a new linter in the future an easy pattern to copy/paste
3. Might there be a "universal exclusion" pattern in lintrunner already that can be leveraged?  And if not, maybe it's worth adding one to reduce the chances of human error down the road?

Reply:
1) More for conciseness. If (ie. if `'fb/**'` is not needed given the inclusions, its noisy to add it imo). I just added what was needed given the inclusion rules as its a good habit to have.
2) Same as 1.
3) I agree, but that change got rejected :P https://github.com/suo/lintrunner/pull/54